markets/bfp-market/contracts/modules/MarginModule.sol

```diff
@@ -598,7 +598,11 @@ contract MarginModule is IMarginModule {
         // When there is no position then we can ignore all running losses/profits but still need to include debt
         // as they may have realized a prior negative PnL.
         if (size == 0) {
-            return marginValues.collateralUsd - Margin.load(accountId, marketId).debtUsd;
+            return
+                MathUtil.max(
```

Comment:
Minor: A quick test for this?

You can use the `__test_addDebtUsdToAccountMargin` function in `TestHelperModule` to quickly add debt to an account.

Reply:
Thanks, this actually exposed a missing cast!